src/gui/dialogs/editor/tod_new_schedule.hpp

```diff
@@ -0,0 +1,50 @@
+/*
+	Copyright (C) 2023 - 2023
+	Part of the Battle for Wesnoth Project https://www.wesnoth.org/
+
+	This program is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation; either version 2 of the License, or
+	(at your option) any later version.
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY.
+
+	See the COPYING file for more details.
+*/
+
+#pragma once
+
+#include "gui/dialogs/modal_dialog.hpp"
+
+namespace gui2
+{
+
+namespace dialogs
+{
+
+/**
+ * @ingroup GUIWindowDefinitionWML
+ *
+ * Dialog that takes new schedule ID and name from the player.
```

Comment:
Mentioning that it doesn't contain the ToD's settings  would be helpful.

Reply:
Added more information.